content/docs/capabilities/ppl.mdx

```diff
@@ -76,6 +76,39 @@ If `nor` is used, the user will have access if their email address does not end
   </div>
 </details>
 
+### Multiple Operators in an Action
+
+You _can_ add multiple operators under the same action block, and it would be valid PPL.
+
+For example, this policy would grant access to anyone with a `family_name: Smith` claim, **or** users with email addresses ending in `domain1` or `domain2`:
+
+```yaml
+allow:
+  and:
+    - claim/family_name: Smith
+  or:
+    - domain:
+        is: domain2.com
+    - domain:
+        is: domain3.com
+```
+
+However, you could write an equivalent policy with multiple `allow` blocks:
+
+```yaml
+- allow:
+    and:
+      - claim/family_name: Smith
+- allow:
+    or:
+      - domain:
+          is: domain1.com
+      - domain:
+          is: domain2.com
+```
+
+Although these policies are equally effective, the second example using multiple actions is the intended method to express PPL.
```

Comment:
(optional)

Another phrasing suggestion (if you think it would be helpful): "For clarity, we recommend using just one operator per rule."